### ISSUE TITLE
Automate rails integration

### ISSUE THREAD

**Issue by onderkalaci** [citusdata/test-automation#65]
As users rely on more rails gem, we should add it to our test automation framework as we add tpch or dml benchmarks.

**Comment by onderkalaci** [comment]
I think we should similarly add django/hibernate integrations

**Comment by furkansahin** [comment]
There are also rails and django codes in citus_docs. We should add them here too.
Skipping for now.
@metdos

**Comment by metdos** [comment]
> Skipping for now.

you're NOT skipping testing them, right :)

**Comment by furkansahin** [comment]
Then making the time needed for documentation at least 2 days. Since I have to create a working environment from scratch for these tools and try to understand what these tools really are and then learn how to use.

**Comment by metdos** [comment]
> Then making the time needed for documentation at least 2 days. Since I have to create a working environment from scratch for these tools and try to understand what these tools really are and then learn how to use.

I see, then you can skip them. Could you add the links you're referring? I think I will add tests to test document and test them. Thanks for raising the issue.

**Comment by furkansahin** [comment]
>I see, then you can skip them. Could you add the links you're referring? I think I will add tests to test document and test them. Thanks for raising the issue.

Of course, you can find the classes in this [url](https://docs.citusdata.com/en/latest/migration/transitioning.html#preparing-to-scale-out-a-multi-tenant-application)
